### ISSUE TITLE
Remove optional automatic STIG Import on app initialization

### ISSUE THREAD

**Issue by cd-rite** [NUWCDIVNPT/stig-manager#1089]
On it's very first initialization, STIGMan can attempt to reach out to DISA to download and import the STIG Compilation library.
This feature is only marginally helpful one time, and is the only place the App initializes an external connection.
To reduce support surface area, it should be removed.

Will need changes to (not necessarily all-inclusive):
utils/config.js
index.js
Envvars - STIGMAN_INIT_IMPORT_STIGS
docs
DockerHub_Readme.md

Basically, anywhere the project references "STIGMAN_INIT_IMPORT_STIGS" or "config.init.importStigs"
